tests/fixtures/5_kethaa.py

```diff
@@ -6,21 +6,43 @@
 @pytest_asyncio.fixture(scope="session")
 async def externally_owned_account_class(starknet: Starknet):
     return await starknet.declare(
-        source="src/kakarot/accounts/eoa/aa/externally_owned_account.cairo",
+        source="src/kakarot/accounts/eoa/externally_owned_account.cairo",
+        cairo_path=["src"],
+    )
+
+
+@pytest_asyncio.fixture(scope="session")
+async def evm_class(starknet: Starknet):
+    return await starknet.declare(
+        source="src/kakarot/accounts/contract/contract_account.cairo",
+        cairo_path=["src"],
+    )
+
+
+@pytest_asyncio.fixture(scope="session")
+async def account_proxy(starknet: Starknet):
+    return await starknet.declare(
+        source="src/kakarot/accounts/proxy/proxy.cairo",
         cairo_path=["src"],
     )
 
 
 @pytest_asyncio.fixture(scope="package")
 async def deployer(
-    starknet: Starknet, externally_owned_account_class, kakarot
+    starknet: Starknet,
+    evm_class,
+    externally_owned_account_class,
+    kakarot,
+    account_proxy,
 ) -> StarknetContract:
     deployer = await starknet.deploy(
-        source="src/kakarot/accounts/eoa/deployer/deployer.cairo",
+        source="src/kakarot/accounts/deployer/deployer.cairo",
```

Comment:
you can `return await starknet(...` the less one defines variable the faster it's faster to catch the purpose of the fixture